src/Mailer/Ownership/SendOwnershipMailCommandHandler.php

```diff
@@ -0,0 +1,141 @@
+    ) {
+        $this->mailer = $mailer;
+        $this->mailsSentRepository = $mailsSentRepository;
+        $this->logger = $logger;
+        $this->organizerRepository = $organizerRepository;
+        $this->identityResolver = $identityResolver;
+        $this->organizerIriGenerator = $organizerIriGenerator;
+        $this->twig = $twig;
+        $this->ownershipSearchRepository = $ownershipSearchRepository;
+    }
+
+    public function handle($command): void
+    {
+        switch (true) {
+            case $command instanceof SendOwnershipRequestedMail:
+                $this->sentMail(
+                    $command,
+                    self::SUBJECT_OWNERSHIP_REQUESTED,
+                    self::TEMPLATE_OWNERSHIP_REQUESTED
+                );
+                break;
+
+        }
+    }
+
+    public function sentMail(SendOwnershipRequestedMail $command, string $rawSubject, string $template): void
+    {
+        $uuid = new Uuid($command->getUuid());
+
+        if ($this->mailsSentRepository->isMailSent($uuid, get_class($command))) {
+            return;
+        }
+
+        try {
+            $ownershipItem = $this->ownershipSearchRepository->getById($uuid->toString());
+        } catch (OwnershipItemNotFound $e) {
+            $this->logger->warning('[ownership-mail] ' . $e->getMessage());
+            return;
+        }
+
+        try {
+            $organizerProjection = $this->organizerRepository->fetch($ownershipItem->getItemId());
+        } catch (DocumentDoesNotExist $e) {
+            $this->logger->warning(sprintf('[ownership-mail] Could not load organizer: %s', $e->getMessage()));
+            return;
+        }
+
+        $organizer = $organizerProjection->getAssocBody();
+
+        //@todo loop over ALL owners of organisation
+        $ownerId = $ownershipItem->getOwnerId();
+        $ownerDetails = $this->identityResolver->getUserById($ownerId);
+
+        if ($ownerDetails === null) {
+            $this->logger->warning(sprintf('[ownership-mail] Could not load owner details for %s', $ownerId));
+            return;
+        }
+
+        $params = [
+            'organisationName' => $organizer['name']['nl'],
+            'firstName' => $ownerDetails->getUserName(),//@todo change to be the correct user
+            'organisationUrl' => $this->organizerIriGenerator->iri($ownershipItem->getItemId()),
+        ];
+
+        $subject = $this->parseSubject($rawSubject, $organizer['name']['nl']);
+        $to = new EmailAddress($ownerDetails->getEmailAddress());
+
+        try {
+            $success = $this->mailer->send(
+                $to,
+                $subject,
+                $this->twig->render($template . '.html.twig', $params),
+                $this->twig->render($template . '.txt.twig', $params),
+            );
+        } catch (LoaderError|RuntimeError|SyntaxError $e) {
+            $this->logger->error($e->getMessage());
+            return;
+        }
+
+        if (!$success) {
+            return;
```

Comment:
I guess it's worth logging that the mail was not send

Reply:
Added extra logs, and also updated the unit tests to verify the log.